src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.html

```diff
@@ -2,8 +2,12 @@
            type="warning"
            *ngIf="displayNotification"
            (close)="close($event)">
-  <div i18n>The Telemetry module is not submitting telemetry data at the
-    moment. Click
+  <div i18n>The
+    <a href="https://docs.ceph.com/en/latest/mgr/telemetry/">Telemetry</a> module is not submitting telemetry data at the
+    moment.
+  <button class="activate-button">
   <a routerLink="/telemetry"
-     class="alert-link">here</a> to activate it now.</div>
+     class="alert-link activate-text">Activate</a></button>
+</div>
```

Comment:
Let's try to be more assertive/emphatic here. We want Ceph users to join the Telemetry program, so why not:

> "The Ceph community needs your help to continue improving: please [activate][button] the [Telemetry](https://docs.ceph.com/en/latest/mgr/telemetry/) module"

Reply:
it would be like this
![Activate-2](https://user-images.githubusercontent.com/56444536/112298133-fcd91400-8c96-11eb-8208-46d1fcadaa7a.png)

Reply:
Looks good!
Does the Activate button link to the 2 steps wizard?

Reply:
@yaarith Yes it does